app/templates/macros/graphs/ockovani.html

```diff
@@ -0,0 +1,169 @@
+{% macro graf_ockovani(data) -%}
+<div id="graf-ockovani"></div>
```

Comment:
Can you add explicit guide to the graph -> click on the legend will turn on or off ?